site/data/docs/custom-app-info.mdx

```diff
@@ -26,3 +26,36 @@ const App = () => {
   );
 };
 ```
+
+### Add a Disclaimer
+
+You can provide your own disclaimer to be displayed at the bottom of the connection modal. The `disclaimer` property in `appInfo` takes a `DisclaimerComponent` with arguments `Text` and `Link`. You can use it to build your own disclaimer that inherits the theming system. We export the `DisclaimerComponent` type for TypeScript users:
+
+```tsx
+import {
+  RainbowKitProvider,
+  DisclaimerComponent,
+} from '@rainbow-me/rainbowkit';
+
+const App = () => {
+  const disclaimer: DisclaimerComponent = ({ Text, Link }) => (
```

Comment:
This needs to be hoisted out of `App` or it will create a new disclaimer component on every render.

Reply:
oops pulling out now